lighthouse-core/test/config/config-test.js

```diff
@@ -365,53 +365,50 @@ describe('Config', () => {
     });
   });
 
-  describe('getAggregations', () => {
+  describe('getCategories', () => {
     it('returns the IDs & names of the aggregations', () => {
-      const runConfig = JSON.parse(JSON.stringify(defaultConfig));
-      const aggs = Config.getAggregations(runConfig);
-      assert.equal(Array.isArray(aggs), true);
-      assert.equal(aggs.length, 5, 'Found the correct number of aggregations');
-      const haveName = aggs.every(agg => agg.name.length);
-      const haveID = aggs.every(agg => agg.id.length);
-      assert.equal(haveName === haveID === true, true, 'they dont have IDs and names');
+      const categories = Config.getCategories(defaultConfig);
```

Comment:
we could maybe move the `JSON.parse(JSON.stringify(defaultConfig))` to a `before()` to make it more consistent per test?